assets/src/i18n/index.js

```diff
@@ -9,8 +9,37 @@ import zhTW from './zh-tw.yaml'
 
 Vue.use(VueI18n)
 
+let locale = (navigator.language || navigator.browserLangugae).toLowerCase()
+switch (true) {
+  case /^en.*/i.test(locale):
+    locale = 'en'
+    break
+  case /^fr.*/i.test(locale):
+    locale = 'fr'
+    break
+  case /^pt.*/i.test(locale):
+    locale = 'pr'
+    break
+  case /^ja.*/i.test(locale):
+    locale = 'ja'
+    break
+  case /^zh-CN/i.test(locale):
+    locale = 'zh-cn'
+    break
+  case /^zh-TW/i.test(locale):
+    locale = 'zh-tw'
+    break
+  case /^zh.*/i.test(locale):
+    locale = 'zh-cn'
+    break
+  default:
+    locale = 'en'
+}
+
+console.log(locale)
```

Comment:
Sorry for being picky, I'll just ask to remove this, please

Reply:
oh, my bad